shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/HomeViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package co.nimblehq.avishek.phong.kmmic.presentation.module
```

Comment:
This is a backend PR so please avoid adding integration-related tasks for integration PR. It is hard to review. 🙏